dem_handler/utils/spatial.py

```diff
@@ -325,3 +334,98 @@ def adjust_bounds_at_high_lat(bounds: BBox) -> tuple:
         bounds = adjust_bounds(bounds, src_crs=4326, ref_crs=3995)
 
     return bounds
+
+
+def resize_bbox(bbox: BoundingBox, scale_factor: float = 1.0) -> BoundingBox:
```

Comment:
I've previously named the argument referring to the BoundingBox as "bounds" rather than bbox. I'm happy with either, but it would be good to be consistent, I think. What do you think?